### PR TITLE
Make new tier changes

### PR DESCRIPTION
**Pull request by JonathanHallam**
Makes tier changes in line with latest guidance

### CODE REVIEW

File: config/local_restrictions.yml [hunk 1]
```diff
@@ -233,18 +260,27 @@ E06000040:
...
+      start_time: "00:01"
 E06000042:
   name: Milton Keynes Council
   restrictions:
     - alert_level: 2
       start_date: 2020-12-02
       start_time: "00:01"
+    - alert_level: 3
+      start_date: 2020-12-19
+      start_time: "00:01"
```
Comment (kevindew) [inline config/local_restrictions.yml:283]: Bad news for @Rosa-Fox

**Review by kevindew — APPROVED** [review]
Syntax looks good to me 👍

File: config/local_restrictions.yml (hunk 1 above)
Comment (Rosa-Fox) [reply config/local_restrictions.yml:283]: noooo!!

**Review by leenagupte — APPROVED** [review]
The date looks good against the spreadsheet. It looks good when run locally too. 👍

**Pull request merged**